seacrowd/sea_datasets/malaysia_tweets/malaysia_tweets.py

```diff
@@ -0,0 +1,164 @@
+editor="Alfred, Rayner
+and Lim, Yuto",
+title="Social Versus Physical Distancing: Analysis of Public Health Messages at the Start of COVID-19 Outbreak in Malaysia Using Natural Language Processing",
+booktitle="Proceedings of the 8th International Conference on Computational Science and Technology",
+year="2022",
+publisher="Springer Singapore",
+address="Singapore",
+pages="577--589",
+abstract="The study presents an attempt to analyse how social media netizens in Malaysia responded to the calls for ``Social Distancing'' and ``Physical Distancing'' as the newly recommended social norm was introduced to the world as a response to the COVID-19 global pandemic. The pandemic drove a sharp increase in social media platforms' use as a public health communication platform since the first wave of the COVID-19 outbreak in Malaysia in April 2020. We analysed thousands of tweets posted by Malaysians daily between January 2020 and August 2021 to determine public perceptions and interactions patterns. The analysis focused on positive and negative reactions and the interchanges of uses of the recommended terminologies ``social distancing'' and ``physical distancing''. Using linguistic analysis and natural language processing, findings dominantly indicate influences from the multilingual and multicultural values held by Malaysian netizens, as they embrace the concept of distancing as a measure of global public health safety.",
+isbn="978-981-16-8515-6"
+}
+"""
+
+_DATASETNAME = "malaysia_tweets"
+_DESCRIPTION = """\
+This tweet data was extracted from tweets in Malaysia based on keywords
+"social distancing" and "physical distancing". We conducted
+sentiment analysis to understand public opinions on health messages
+during the COVID-19 pandemic. Tweets from January 2020 to July 2021
+were extracted using Python module snscrape and sentiments were obtained
+automatically using Polyglot and MALAYA NLP tools due to multilingual data.
+"""
+
+_HOMEPAGE = "https://github.com/sarahjuan/malaysia-tweets-with-sentiment-labels"
+
+_LANGUAGES = ["zlm,","eng"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.UNKNOWN.value # example: Licenses.MIT.value, Licenses.CC_BY_NC_SA_4_0.value, Licenses.UNLICENSE.value, Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {
+    _DATASETNAME: "https://raw.githubusercontent.com/sarahjuan/malaysia-tweets-with-sentiment-labels/main/data/cleaned_tweets_sentiments.csv",
+}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+# TODO: set this to a version that is associated with the dataset. if none exists use "1.0.0"
+#  This version doesn't have to be consistent with semantic versioning. Anything that is
+#  provided by the original dataset as a version goes.
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class MalaysiaTweets(datasets.GeneratorBasedBuilder):
```

Comment:
The class name should be `MalaysaiTweetsDataset`. Please refer to `https://github.com/SEACrowd/seacrowd-datahub/blob/master/DATALOADER.md` for the full instructions.